lib/vault/api/auth.rb

```diff
@@ -287,11 +287,13 @@ def gcp(role, jwt, path = 'gcp')
     #   The path to the auth backend to use for the login procedure.
     #
```

Comment:
Might want to add an `@param` comment for the name parameter? Not sure if these comments are used to generate docs.

Reply:
Added in d7586f0.